apps/mobile/src/app/wallet/_layout.tsx

```diff
@@ -100,7 +103,27 @@ export default function StackLayout() {
       <Stack.Screen name="create-new-wallet" options={{ header: () => NavigationBackSimple }} />
       <Stack.Screen name="recover-wallet" options={{ header: () => NavigationBackSimple }} />
       <Stack.Screen name="secure-your-wallet" options={{ header: () => NavigationBackSimple }} />
-      <Stack.Screen name="settings/index" options={{ header: () => NavigationSettings }} />
+      <Stack.Screen name="settings/index" options={{ header: () => NavigationSettings() }} />
```

Comment:
Just to be clear `settings` are global settings and `wallet-settings` are wallet specific ? Or can they be combined.

Is the name `name=` just a label? I'm wondering if it would make sense to co-locate with the route consts

Reply:
I see your TODO below now to refactor `wallet-settings` 👍

Reply:
I'll clean this up in the next phase of this work.